docs/Reference/Evm-Tool.md

```diff
@@ -323,10 +323,27 @@ Set to true for EVMLab Fuzzing.
 Whether or not `json` is set, a summary JSON object is printed to standard output for each
 state test executed.
 
+#### `nomemory`
+
+=== "Syntax"
+
+    ```bash
+    --nomemory=<boolean>
+    ```
+
+=== "Example"
+
+    ```bash
+    --nomemory=true
+    ```
+
+By default, when tracing operations the memory is traced for each operation.
+For memory heavy scripts, setting this option may reduce the volume of JSON output.
```

Comment:
```suggestion
For memory heavy scripts, setting this option to `true` may reduce the volume of JSON output.
```
Can you also add backticks to the instances of "true" in the description of the previous `json` option?